libr/io/p/io_self.c

```diff
@@ -131,7 +132,32 @@ static int update_self_regions(RIO *io, int pid) {
 	return bsd_proc_vmmaps(io, pid);
 #else
 #ifdef _MSC_VER
-#pragma message ("Not yet implemented for this platform")
+	int perm;
+	MEMORY_BASIC_INFORMATION mbi;
+	HANDLE h = OpenProcess (PROCESS_QUERY_INFORMATION | PROCESS_VM_READ, 0, pid);
+	size_t size = VirtualQuery (0, &mbi, sizeof (mbi));
+	char *name = calloc (1024, sizeof (char));
+	while (size) {
+		ut64 to = (ut8 *) mbi.BaseAddress + mbi.RegionSize;
+		GetMappedFileName (h, (LPVOID) mbi.BaseAddress, name, 1024);
```

Comment:
check return value, otherwise name can be not initialized. also are we sure that this function null terminates the buffer always? otherwise we should memset instead of just name[0]=0

Reply:
see comment below